app.rb

```diff
@@ -2,74 +2,12 @@
 require 'sinatra/namespace'
 require 'sinatra/respond_with'
 
+require_relative './routes/raw'
+require_relative './routes/analysis'
+require_relative './routes/engine'
+
 require_relative './lib/config'
-require_relative './lib/morpheus'
-require_relative './lib/parser'
-require_relative './lib/error'
 
 set :protection, except: [:path_traversal]
+set :strict_paths, false
 set :port, Config::PORT
-
-namespace '/raw' do
```

Comment:
Most of the files changed in this PR are a changed as a result of splitting this up into multiple files. The changes relevant to the engine functionality are:

* `lib/engine.rb`
* `lib/engine_list.rb`
* `routes/engine.rb`